test/cli/workspace/create/args

```diff
@@ -0,0 +1 @@
+remote workspace create --storage-api-token %%TEST_KBC_STORAGE_API_TOKEN%% --name foo --type snowflake --size small
```

Comment:
prazdny riadok na konci by potesil, ale to vyriesi linter, ... takze to nemusis upravovat (ale mozes si prestavit IDE).

Reply:
Nastaveno, jen jsem si všiml, že občas mi to přidá dokonce dvě newlines na konci. To pak ještě dořeším.